Specs/main.js

```diff
@@ -0,0 +1,59 @@
+function random() {
+  const ran = (Math.floor(Math.random() * 100) +1)
+  console.log(ran)
+  let userInput = prompt("What's your guess of the number?", "")
+  let guessNum = parseInt(userInput)
+
+  if(guessNum === ran) {
+    alert("you won!")
+  } else if(guessNum < ran) {
+    alert("your guess is too small")
+  } else if(guessNum > ran) {
+    alert("your guess is too high")
+  }
+
+  if(userInput === "exit") {
+    return null
+  }
+}
+
+
+var splitWords;
+var checkIndex;
+var num;
+
+function stringIndices() {
+  let wordsToParse = prompt("Enter words to parse", "")
+  let regEx = wordsToParse.replace(/[W_]/g, " ")
+  let change = document.getElementById("change").innerHTML = wordsToParse
```

Comment:
This line too, is a bit confusing. It seems you are declaring a variable, change, and assigning it a value that is also assigning a new value to itself. If this works awesome 😁 , it could be I just haven't seen this before.